pinot-spi/src/main/java/org/apache/pinot/spi/stream/PartitionGroupConsumptionStatus.java

```diff
@@ -35,15 +35,18 @@
  * This information is needed by the stream, when grouping the partitions/shards into new partition groups.
  */
 public class PartitionGroupConsumptionStatus {
+
+  private final String _topicName;
```

Comment:
topicName seems to be a notNull value, let's verify this carefully in the integration tests.